src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -71,6 +73,17 @@ export interface BankAccountSelection {
 }
 
 export const PaymentRoute: FC<PaymentRouteProps> = props => {
+  const { state, dispatch } = useContext(PaymentContext)
```

Comment:
```tsx
const { state, dispatch } = useOrderPaymentContext() 
```

Reply:
yup! I'm secretly hoping that testing would also be easier with a hook.